dcm4che-imageio-rle/src/main/java/org/dcm4che3/imageio/plugins/rle/RLEImageReader.java

```diff
@@ -242,6 +243,7 @@ private void read(byte[][] bands) throws IOException {
 
     private void read(short[] data) throws IOException {
         readRLEHeader(2);
+        Arrays.fill(data, (short) 0);
```

Comment:
Hi Gunter,
thanks for the fix! Shouldn't the null filling also be done in the read(byte[][] bands) method?